tests/test-sessions/test-mongodb-sessions/src/test/java/org/eclipse/jetty/nosql/mongodb/MongoTestHelper.java

```diff
@@ -269,6 +268,51 @@ public static void createSession (String id, String contextPath, String vhost,
         sets.put(MongoSessionDataStore.__ACCESSED, accessed);
         sets.put(MongoSessionDataStore.__LAST_ACCESSED, lastAccessed);
 
+        if (attributes != null)
+        {
+            SessionData tmp = new SessionData (id, contextPath, vhost, created, accessed, lastAccessed, maxIdle, attributes);
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            ObjectOutputStream oos = new ObjectOutputStream(baos);
+            SessionData.serializeAttributes(tmp, oos);
```

Comment:
`ObjectOutputStream` not closed, use try-with-resources.

Reply:
ok